azavea/azavea-branding-guide
# Add copy/paste functionality

## Overview
This PR adds copy/paste functionality to blocks of text. The button changes to say "Copied" when clicked.

### Demo
<img width="773" alt="screen shot 2018-08-16 at 11 22 22 am" src="https://user-images.githubusercontent.com/5672295/44218003-bf0ec900-a146-11e8-8c7e-c5852058c5be.png">
<img width="821" alt="screen shot 2018-08-16 at 11 22 26 am" src="https://user-images.githubusercontent.com/5672295/44218004-bf0ec900-a146-11e8-941c-3e22bcfb005b.png">


### Notes
- Using the StyleGuideGuide's JS plugin: https://clipboardjs.com/

## Testing Instructions
- `git pull`
- Go to the Identity > Company page
- Click on a "Copy" button
- You should now have the copied text in your clipboard
- Confirm that this works for other copy/paste components on this page and Fellowships, Products, Open Source, and Community.

Closes #2
